Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -227,6 +227,14 @@ describe('DataSources/KmlDataSource', function() {
         });
     });
 
+    it('load works with a KMZ URL with Windows-style paths', function() {
+        var dataSource = new KmlDataSource(options);
+        return dataSource.load('Data/KML/backslash.kmz').then(function(source) {
+            expect(source).toBe(dataSource);
+            expect(source.entities.values[0]._billboard._image._value).toBeDefined();
```

Comment:
This test also passes on master so it will need to be tweaked.

I noticed that `source.entities.values[0]._billboard._image._value.url` is a data uri if it successfully loads the image. You can use `isDataUri` to check that url is in fact a data uri.

Reply:
This issue was resolved by using `isDataUri`. The test fails on master, but passes on this branch.